deployment/dockerfiles/dev/event-listener

```diff
@@ -2,32 +2,25 @@
 
 FROM node:9 as origin-js-build
 
-# Invalidate Docker cache if necessary by adding the hash of the current commit
-# from the remote repository
-ADD https://api.github.com/repos/OriginProtocol/origin/git/refs/heads/master /version.json
-# Clone origin-js from master (development) branch
-RUN git clone --depth 1 --branch master https://github.com/OriginProtocol/origin.git /app
-
-WORKDIR /app/origin-js
+COPY ./origin-js /app
+WORKDIR /app
 
+ENV NODE_ENV=production
 RUN npm install --quiet --no-progress
 RUN npm run build
 
 # --- Event listener
 
 FROM node:9 as build
 
+COPY ./origin-discovery /app
 WORKDIR /app
 
-# Invalidate Docker cache if necessary
-COPY --from=origin-js-build /version.json /
-COPY --from=origin-js-build /app/origin-discovery/ /app/
-
-RUN npm install --quiet --no-progress
-
+ENV NODE_ENV production
```

Comment:
nit: duplicate since already defined on line #8, so could be removed.
same comment for other dockerfiles that have same pattern.